Cargo.toml

```diff
@@ -348,7 +348,7 @@ mesc = "0.3"
 memchr = "2.7"
 num-format = "0.4"
 parking_lot = "0.12"
-proptest = "1.8.0"
+proptest = "1.9.0"
```

Comment:
note, we may want to leave this bump out of this PR

was released 2 months ago https://crates.io/crates/proptest/1.9.0 but may require special handling / review